daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/NodeInfo.scala

```diff
@@ -89,11 +89,11 @@ object NodeInfo {
 
     final def requiredAuthorizers: Set[Party] =
       // TODO https://github.com/digital-asset/daml/issues/8020
-      sys.error("rollback nodes are not supported")
+      Set.empty
 
     final def informeesOfNode: Set[Party] =
       // TODO https://github.com/digital-asset/daml/issues/8020
-      sys.error("rollback nodes are not supported")
+      Set.empty
```

Comment:
This is obviously broken but it is good enough to fix the scenario service (who doesn’t use those methods). I would propose to merge this and then fix it in a separate PR.